ui/pages/confirmations/confirm-transaction/confirm-transaction.component.js

```diff
@@ -129,7 +127,7 @@ const ConfirmTransaction = () => {
     startPolling: (input) =>
       gasFeeStartPollingByNetworkClientId(input.networkClientId),
     stopPollingByPollingToken: gasFeeStopPollingByPollingToken,
-    input: { networkClientId: transaction.networkClientId ?? networkClientId },
```

Comment:
Is this more related to removing use of a globally selected network client? Should this be separated out?

Reply:
It's required to for the confirmation navigation to function across confirmations on different chains.

Reply:
🤔 if the transaction has the networkClientId and we're using that if present why do we have to remove use of the "selectedNetworkClientId" in this context? I'm not against removing it, just trying to understand why its necessary for this change

Reply:
Fair point, there are global selector removals in this PR that are necessary to support navigation between confirmations on different chains, but this example is unique in that it prioritises the transaction first. So just a force of habit to remove as it was a small one.